<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>457</width>
    <height>394</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,6">
       <item>
        <widget class="QFrame" name="frame_2">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="metodosLabel">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>160</width>
              <height>40</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>METODOS:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="puntofijoButton">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Punto Fijo</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="biseccionButton">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Bisecci√≥n</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="newtonButton">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Newton-Raphson</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="secanteButton">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Secante</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="difeButton">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Diferencias divididas</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="jacobiButton">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Jacobi</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="gaussButton">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Gauss-Seidel</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="trapecioButton">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Trapecio</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="simpsonButton">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Simpson</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_3">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>1</number>
          </property>
          <property name="topMargin">
           <number>1</number>
          </property>
          <property name="rightMargin">
           <number>1</number>
          </property>
          <property name="bottomMargin">
           <number>1</number>
          </property>
          <item>
           <widget class="QStackedWidget" name="stackedWidget">
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="QWidget" name="page_punto_fijo">
             <layout class="QGridLayout" name="gridLayout">
              <item row="0" column="0">
               <layout class="QGridLayout" name="gridLayout_2">
                <item row="0" column="0">
                 <layout class="QVBoxLayout" name="verticalLayout_5">
                  <item>
                   <layout class="QGridLayout" name="gridLayout_3">
                    <item row="0" column="0">
                     <widget class="QLabel" name="label_2">
                      <property name="font">
                       <font>
                        <pointsize>12</pointsize>
                       </font>
                      </property>
                      <property name="text">
                       <string>f(x):</string>
                      </property>
                     </widget>
                    </item>
                    <item row="0" column="2">
                     <widget class="QPushButton" name="funcionButton">
                      <property name="minimumSize">
                       <size>
                        <width>30</width>
                        <height>30</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>66</width>
                        <height>35</height>
                       </size>
                      </property>
                      <property name="text">
                       <string/>
                      </property>
                      <property name="icon">
                       <iconset>
                        <normaloff>../../Downloads/calculadora.png</normaloff>../../Downloads/calculadora.png</iconset>
                      </property>
                      <property name="iconSize">
                       <size>
                        <width>28</width>
                        <height>28</height>
                       </size>
                      </property>
                     </widget>
                    </item>
                    <item row="0" column="1">
                     <widget class="QLineEdit" name="fx_lineEdit"/>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
              <item row="1" column="0">
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <widget class="QLabel" name="label_4">
                  <property name="font">
                   <font>
                    <pointsize>12</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>g(x):</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="gx_lineEdit"/>
                </item>
                <item>
                 <widget class="QPushButton" name="funcionButton2">
                  <property name="minimumSize">
                   <size>
                    <width>30</width>
                    <height>30</height>
                   </size>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="icon">
                   <iconset>
                    <normaloff>../../Downloads/calculadora.png</normaloff>../../Downloads/calculadora.png</iconset>
                  </property>
                  <property name="iconSize">
                   <size>
                    <width>28</width>
                    <height>28</height>
                   </size>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="4" column="0">
               <layout class="QHBoxLayout" name="horizontalLayout_6">
                <property name="spacing">
                 <number>4</number>
                </property>
                <property name="sizeConstraint">
                 <enum>QLayout::SetDefaultConstraint</enum>
                </property>
                <item>
                 <widget class="QLabel" name="result_Label">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>300</width>
                    <height>40</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <pointsize>12</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>Solucion:</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="2" column="0">
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <item>
                 <widget class="QLabel" name="label_5">
                  <property name="font">
                   <font>
                    <pointsize>12</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>Xo</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="xo_lineEdit"/>
                </item>
                <item>
                 <widget class="QLabel" name="label_6">
                  <property name="font">
                   <font>
                    <pointsize>12</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>Tolerancia</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="tol_lineEdit"/>
                </item>
               </layout>
              </item>
              <item row="3" column="0">
               <layout class="QHBoxLayout" name="horizontalLayout_5">
                <item>
                 <widget class="QPushButton" name="calcularButton">
                  <property name="maximumSize">
                   <size>
                    <width>160</width>
                    <height>30</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <pointsize>12</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>Calcular</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="5" column="0">
               <layout class="QHBoxLayout" name="horizontalLayout_7">
                <item>
                 <widget class="QPushButton" name="graf_button">
                  <property name="maximumSize">
                   <size>
                    <width>160</width>
                    <height>30</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <pointsize>12</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>Ver Gr√°fica</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="6" column="0">
               <layout class="QHBoxLayout" name="horizontalLayout_9">
                <item>
                 <widget class="QPushButton" name="tab_button">
                  <property name="maximumSize">
                   <size>
                    <width>160</width>
                    <height>30</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <pointsize>12</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>Ver tabla</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="page_biseccion">
             <widget class="QLabel" name="label_3">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>10</y>
                <width>35</width>
                <height>10</height>
               </rect>
              </property>
              <property name="text">
               <string>biseccion</string>
              </property>
             </widget>
            </widget>
            <widget class="QWidget" name="page_newton"/>
            <widget class="QWidget" name="page_secante"/>
            <widget class="QWidget" name="page_diferencias"/>
            <widget class="QWidget" name="page_jacobi"/>
            <widget class="QWidget" name="page_gauss"/>
            <widget class="QWidget" name="page_trapecio"/>
            <widget class="QWidget" name="page_simpson"/>
            <widget class="QWidget" name="page_calculadora">
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <widget class="QLabel" name="label">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>300</width>
                  <height>40</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <pointsize>14</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Ingrese la funcion:</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QGridLayout" name="gridLayout_4">
                <item row="1" column="3">
                 <widget class="QPushButton" name="inicio_button">
                  <property name="minimumSize">
                   <size>
                    <width>25</width>
                    <height>30</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>(</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1" colspan="3">
                 <widget class="QLineEdit" name="funcion_Label">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>30</height>
                   </size>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QPushButton" name="euler_button">
                  <property name="minimumSize">
                   <size>
                    <width>25</width>
                    <height>30</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>e</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1">
                 <widget class="QPushButton" name="resta_button">
                  <property name="minimumSize">
                   <size>
                    <width>25</width>
                    <height>30</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>-</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QPushButton" name="suma_button">
                  <property name="minimumSize">
                   <size>
                    <width>25</width>
                    <height>30</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>+</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="3">
                 <widget class="QPushButton" name="fin_button">
                  <property name="minimumSize">
                   <size>
                    <width>25</width>
                    <height>30</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>)</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="2">
                 <widget class="QPushButton" name="mult_button">
                  <property name="minimumSize">
                   <size>
                    <width>25</width>
                    <height>30</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>x</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="0">
                 <widget class="QPushButton" name="div_button">
                  <property name="minimumSize">
                   <size>
                    <width>25</width>
                    <height>30</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>/</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QPushButton" name="pi_button">
                  <property name="minimumSize">
                   <size>
                    <width>25</width>
                    <height>30</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>œÄ</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="2">
                 <widget class="QPushButton" name="ln_button">
                  <property name="minimumSize">
                   <size>
                    <width>25</width>
                    <height>30</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>ln</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="2">
                 <widget class="QPushButton" name="cos_button">
                  <property name="minimumSize">
                   <size>
                    <width>25</width>
                    <height>30</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>cos</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="1">
                 <widget class="QPushButton" name="sen_button">
                  <property name="minimumSize">
                   <size>
                    <width>25</width>
                    <height>30</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>sen</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="3">
                 <widget class="QPushButton" name="next_button">
                  <property name="minimumSize">
                   <size>
                    <width>25</width>
                    <height>30</height>
                   </size>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="icon">
                   <iconset>
                    <normaloff>../../Downloads/aceptar.png</normaloff>../../Downloads/aceptar.png</iconset>
                  </property>
                 </widget>
                </item>
                <item row="0" column="0">
                 <widget class="QComboBox" name="funcion_ComboBox">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>30</height>
                   </size>
                  </property>
                  <item>
                   <property name="text">
                    <string>f(x):</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>g(x):</string>
                   </property>
                  </item>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="page_grafica">
             <widget class="QLabel" name="label_8">
              <property name="geometry">
               <rect>
                <x>50</x>
                <y>30</y>
                <width>35</width>
                <height>10</height>
               </rect>
              </property>
              <property name="text">
               <string>Grafica:</string>
              </property>
             </widget>
            </widget>
            <widget class="QWidget" name="page_tabla">
             <widget class="QLabel" name="label_9">
              <property name="geometry">
               <rect>
                <x>40</x>
                <y>30</y>
                <width>35</width>
                <height>10</height>
               </rect>
              </property>
              <property name="text">
               <string>Tabla:</string>
              </property>
             </widget>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
